packages/main/src/MenuItem.ts

```diff
@@ -5,6 +5,7 @@ import i18n from "@ui5/webcomponents-base/dist/decorators/i18n.js";
 import { isPhone } from "@ui5/webcomponents-base/dist/Device.js";
 import AriaHasPopup from "@ui5/webcomponents-base/dist/types/AriaHasPopup.js";
 import type { AccessibilityAttributes } from "@ui5/webcomponents-base/dist/types.js";
+import { renderFinished } from "@ui5/webcomponents-base/dist/Render.js";
```

Comment:
why do we need render finish?

Reply:
So we ensure that before focusing the first item, the control is rendered. If it's removed, the focus functions wont work.

Reply:
Can you give me an example where it is not working. 

In theory if `focus` is called on element which is defined it internally should wait for its rendering to finish.